deploy/handler/operator.yaml

```diff
@@ -182,6 +182,10 @@ spec:
   selector:
     matchLabels:
       name: {{template "handlerPrefix" .}}nmstate-handler
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 10%
```

Comment:
kubernetes-nmstate default is 50%, any reason to not keep the same?

Reply:
This is after update right?

Reply:
I was following the recommended default recommended in https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#upgrade-and-reconfiguration without giving it a further thought.

This is for the upgrade only AFAIK.

I don't fully follow what is the 50% used for in e2e, do you recall?

Reply:
> I was following the recommended default recommended in https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#upgrade-and-reconfiguration without giving it a further thought.

Aaah, of course, I got it. At first I thought that this is somehow setting the NNCP defauld of maxUnavailable, but this is a different maxUnavailable :).

> I don't fully follow what is the 50% used for in e2e, do you recall?
Not sure what you mean, but as I said, the 50% I pointed to is related to NNCP rollout, not handler upgrade :)

/lgtm
/approve

Feel free to unhold if all is clear
/hold

Reply:
Thanks o/